src/helpers/agent.ts

```diff
@@ -24,9 +24,9 @@ import { WebSocket } from 'ws'
 class CustomFileSystem extends NodeFileSystem {
   public constructor() {
     super({
-      baseDataPath: `${process.cwd()}/credo/data`,
-      baseCachePath: `${process.cwd()}/credo/cache`,
-      baseTempPath: `${process.cwd()}/credo/tmp`,
+      baseDataPath: `/credo/data`,
+      baseCachePath: `/credo/cache`,
```

Comment:
what's the plan for caching again? are we caching locally? The native indy-vdr cache should handle that. If instead we're using a shared cache stored in the openshift deployment I have some security concerns about cache poisoning I want to make sure are addressed

Reply:
The proxy server only does client side caching in the wallet, which is configurable there. This cache directory doesn't get used, only the data directory gets used. I just added all the available options for base paths because when agents get initialized they create three empty dirs with these paths.